src/NUnitFramework/framework/Attributes/TestFixtureSourceAttribute.cs

```diff
@@ -126,7 +122,7 @@ public IEnumerable<TestSuite> BuildFrom(Type type, IPreFilter filter)
         {
             Type sourceType = SourceType ?? type;
 
-            foreach (TestFixtureParameters parms in GetParametersFor(sourceType))
+            foreach (ITestFixtureData parms in GetParametersFor(sourceType))
```

Comment:
Thanks so much for doing this! This has bugged me for a while. I thought it was due to having to use the type name instead of `var` which silently introduces a possible invalid cast exception when the method return type is changed.